pytest.ini

```diff
@@ -1,2 +1,8 @@
 [pytest]
-norecursedirs = .* dist CVS _darcs {arch} *.egg
\ No newline at end of file
+DJANGO_SETTINGS_MODULE = autoreduce_webapp.test_settings
```

Comment:
Is it strictly necessary that DJANGO_SETTINGS_MODULE is defined here? Could it be defined just in test_admin.py?